templates/flutter/lib/client.dart.twig

```diff
@@ -86,8 +86,10 @@ class Client {
             final Directory cookieDir = await _getCookiePath();
             cookieJar = new PersistCookieJar(dir:cookieDir.path);
             this.http.interceptors.add(CookieManager(cookieJar));
-            PackageInfo packageInfo = await PackageInfo.fromPlatform();
-            addHeader('Origin', 'appwrite-' + type + '://' + packageInfo.packageName);
+            if(Platform.isAndroid || Platform.isAndroid) {
```

Comment:
You're checking for Android twice, I think it will be better to skip this check only for the Web platform. All other should report the origin header.